templates/test/cluster-template-prow-ha.yaml

```diff
@@ -204,7 +204,7 @@ spec:
       image:
         checksum: ${IMAGE_RAW_CHECKSUM}
         checksumType: ${IMAGE_CHECKSUM_TYPE}
-        format: ${IMAGE_FORMAT}
+        format: raw
```

Comment:
why are we reverting it back? if that is the case, you must get rid of the `IMAGE_FORMAT` var from environment.sh

Reply:
Yup, I realized that I need to remove IMAGE_FORMAT from environment.sh as well, just didn't push it yet.
As for why: since I use the variables `IMAGE_RAW_URL` and `IMAGE_RAW_CHECKSUM`, the image can be always assumed to use the raw format. Changing this will require modifying the template here anyway, so I think it's better not to give a false sense of the image format being changeable by `environment.sh` alone.